package.json

```diff
@@ -800,7 +806,8 @@
     "pretty-ms": "^5.0.0",
     "proxy": "^2.1.1",
     "q": "^1.5.1",
-    "react-test-renderer": "^16.12.0",
+    "react-test-renderer": "^16.14.0",
+    "react-textarea-autosize": "^8.3.4",
```

Comment:
Is this used?

Reply:
Looks like `react-textarea-autosize` was only used for type resolution in the original PR and can probably be removed in `main` as well.